frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/selectors/template.ts

```diff
@@ -54,6 +54,7 @@ export const getTemplateValidation = (state, id: string): TemplateValidations =>
   const templateValidations = getTemplateValidations(state, id);
   if (templateValidations && templateValidations.length > 0) {
     templateValidations.length > 1 &&
+      // eslint-disable-next-line no-console
       warn('WARNING: getTemplateValidation: multiple template validations detected!');
```

Comment:
you can put the whole console statement here now

Reply:
done.